frontend/public/locales/en/translation.json

```diff
@@ -1570,6 +1574,7 @@
   "Imported clusters cannot be destroyed.": "Imported clusters cannot be destroyed.",
   "importing": "Importing",
   "Importing": "Importing",
+  "In OCM policies, Response action refers to the remediationAction. For Gatekeeper, `remediationAction` is referred to as `enforcementAction.`": "In OCM policies, Response action refers to the remediationAction. For Gatekeeper, `remediationAction` is referred to as `enforcementAction.`",
```

Comment:
```suggestion
  "In OCM policies, response action refers to the remediationAction. For Gatekeeper, response action refers to the enforcementAction.": "In OCM policies, response action refers to the remediationAction. For Gatekeeper, response action refers to the enforcementAction.",
```

Reply:
Can you remind me what OCM is? Is it Red Hat OpenShift Cluster Manager or Open Cluster Management?

Reply:
It's Open Cluster Management, and that raises the point that we should probably spell it out. I'll update the suggestion!

Reply:
Why are you using backticks in UI string? If you want something formatted as code in the UI, you will need to use a `<Trans>` component supply the formatting yourself.

Reply:
@KevinFCormier  Thanks a lot Kevin! We've decided not to use that formatting.